src/views/Question.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import { Button } from "react-bootstrap";
+import styled from "styled-components";
+import graphql from 'babel-plugin-relay/macro'
+
+import Query from '../components/Query'
+import QuestionCard from "../components/QuestionCard";
+import SubmitAnswerForm from "../components/SubmitAnswerForm";
+import AnswerCard from "../components/AnswerCard";
+
+const query = graphql`
+  query QuestionQuery ($question_id: ID!){
+    node(id: $question_id) {
+      ...QuestionCard_question
+      ...on QuestionNode {
+        answers { edges { node { id ...AnswerCard_answer}}}
+      }
+    }
+  }
+`
+
+export default function Question(props) {
```

Comment:
All functions should have a space between the arguments and the function name as per Standard JS. i.e.:
```javascript
function Question (props) {}
```